src/odemis/gui/cont/tabs.py

```diff
@@ -1716,6 +1716,16 @@ def __init__(self, name, button, panel, main_frame, main_data):
         self._settings_controller.setter_temporalspec_bck_file = self.set_temporalspec_background
         self._settings_controller.setter_spec_file = self.set_spec_comp
 
+        if main_data.role is None:
+            # HACK: Move the logo to inside the FileInfo bar, as the tab buttons
+            # are hidden when only one tab (ie, no backend).
```

Comment:
when only one tab is selected or when there is only one tab?

Reply:
Only one tab shown. -> Fixed comment.